tools/vscode-azurewebpubsub/src/workflows/service/create/steps/InputSerivceSkuUnitCountStep.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { KnownWebPubSubSkuTier } from "@azure/arm-webpubsub";
+import { AzureWizardPromptStep, IAzureQuickPickItem } from "@microsoft/vscode-azext-utils";
+import { pricingLink, SKU_TIER_TO_UNIT_COUNT_LIST } from "../../../../constants";
+import { localize } from "../../../../utils";
+import { ICreateServiceContext } from "../ICreateServiceContext";
+
+
+export class InputSerivceSkuUnitCountStep extends AzureWizardPromptStep<ICreateServiceContext> {
+    public async prompt(context: ICreateServiceContext): Promise<void> {
+        if (!(context.Sku?.sku)) {
+            throw new Error("Invalid context or sku");
+        }
+
+        var picks: IAzureQuickPickItem<number>[] = [];
+        const tier: string | undefined = context.Sku.sku.tier
+        switch (tier) {
+            case KnownWebPubSubSkuTier.Free:
```

Comment:
no need for Free

Reply:
also provide a default value for other tiers?

Reply:
the pick prompt for user will be automatically skipped if there is only one choice. Standard_S1 and Premium_P1 already has a default unit value 1. Default unit count for Premium_P2 is 100.